tests/test_demes.py

```diff
@@ -1,9 +1,10 @@
 import unittest
 import copy
 import pathlib
-import json
 
-import jsonschema
+# import json
```

Comment:
delete?

Reply:
You're right, it's probably better to just delete this. I thought maybe we'd reuse it again soon for validating against the spec, but probably we'll do something different (via CI) to compare with the spec and/or spec repo examples.